<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeicalBossWidget</class>
 <widget class="QWidget" name="SpeicalBossWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1357</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="JueRadioButton">
      <property name="text">
       <string>角</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="RoverRadioButton">
      <property name="text">
       <string>无妄者</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="HecateRadioButton">
      <property name="text">
       <string>赫卡忒</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>110</y>
     <width>346</width>
     <height>456</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>捡声骸方式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pickUpEchoMethod">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>方形</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>普通形</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>直线形</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>蚊香形</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>声骸拾取范围</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pickUpEchoRange">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>蚊香形拾取范围</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spiralPickupRange">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="isPressFwhenFighting">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>战斗时按F</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>自选副本等级</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dungeonLevel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>动作额外延迟</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="actionDalay">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>毫秒</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="jumpAttackToHeal">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>2号位 跳A回血</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>吸收文字相似度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="absorbThres">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>大招检测方式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ultimateSkillDetectMethod">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>方式一</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>方式二</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="bossHPAssis">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>BOSS血条辅助判定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isTriggerNoSwitch">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>开启不切人战斗</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>开启时机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="triggerNoSwitchCondition">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>0.125</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.250</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.375</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.625</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="startSingleBoss">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>启动脚本</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>500</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>要求
窗口模式 1280 720分辨率
如提示不对，可将windows显示设置 dpi缩放改为100%
然后重启脚本重启游戏试试
必须关闭所有滤镜和画质MOD
如果担心电脑卡可设置最低画质30帧
游戏不可最小化但可放在后台被遮盖
在能操作人物进行普攻移动时
点击开始按钮启动脚本
禁止使用角色油壶 大招太长
1号位任意 2号位奶妈推荐维里奈
3号位必须是成年人体型角色 3号不能是椿
角色必须是快走状态（CTRL切换）
背包图标（地图旁边）不能有红色叹号
脚本虽然理论上是后台运行，但会抢鼠标
建议挂机时看视频
ALT+F12可停止脚本运行
选中的boss必须开通了快速旅行！！
现在有复活检测 需要背包至少有一个复活药
有的声骸没捡到没关系的挂一晚上都有了</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>进入你左上角选中的boss副本
已经准备和BOSS干架了
然后点击启动脚本
如需更换boss 先停止后台
更换boss后再次启动脚本</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
